### DISCUSSION TITLE
How do I change the index page of my profile?

### DISCUSSION THREAD

**Discussion by SapoPretoDaChuva** [community/community#18416] [Other Feature Feedback, Questions, & Ideas]
please

**Comment by ale-lit** [comment]
If you add a README file to the root of a public repository with the same name as your username, that README will automatically appear on your profile page.

More info: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile

**Comment by Akalanka1337** [comment]
Hello,
Please follow these steps.
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme

**Comment by trinib** [comment]
> please

@SapoPretoDaChuva Add file named README.md and edit
